doc/source/data/dataset.rst

```diff
@@ -173,159 +173,6 @@ Advanced users can refer directly to the Ray Datasets :ref:`API reference <data-
         :text: Check Out Other Data Processing Options
         :classes: btn-outline-info btn-block
 
-
-.. _data-compatibility:
-
-------------------------
-Datasource Compatibility
-------------------------
-
-Ray Datasets supports reading and writing many file formats.
-The following compatibility matrices will help you understand which formats are currently available.
-
-If none of these meet your needs, please reach out on `Discourse <https://discuss.ray.io/>`__ or open a feature
-request on the `Ray GitHub repo <https://github.com/ray-project/ray>`__, and check out
-our :ref:`guide for implementing a custom Datasets datasource <datasets_custom_datasource>`
-if you're interested in rolling your own integration!
-
-Supported Input Formats
```

Comment:
Have you considered just linking this section to https://docs.ray.io/en/master/data/api/input_output.html and keeping the above paragraph largely intact? It seems valuable to have a list of dataset formats linked in the landing page (but deduping as you are proposing is a good idea) :)